docs/Helix.md

```diff
@@ -36,8 +51,6 @@ There's a link embedded in the build.cmd log of the helix target on Azure Pipeli
 
 The link will take you to an overview of all the tests with clickable links to the logs and each run broken down by queue.
 
-All of the helix runs for aspnetcore can be found here <https://mc.dot.net/#/user/aspnetcore/builds>.
```

Comment:
Is there no longer a page from which it's possible to navigate to specific runs❔ I'm wondering if the https://helix.dot.net/ URIs mentioned above and the required cut-and-pasting is the best we can do when the URIs aren't available from the build logs.

Reply:
I don't believe so, azdo ui is the main view since you can see all the failing tests and its pretty good now getting to all the files directly from the failing test.  But sometimes if you want to get the raw payloads and dig around more, spelunking around the underlying urls for all the workitems is useful